lm_eval/tasks/ds1000.py

```diff
@@ -99,9 +101,15 @@ def get_prompt(self, doc):
         Builds the prompt for the LM to generate from.
         :param doc: dict[str: str]
             sample from the test dataset
-        :return: str
+        :return: str | dict[str: str]
         """
-        return doc["prompt"]
+        if self._mode == "Completion":
+            return doc["prompt"]
+        elif self._mode == "Insertion":
+            prefix, suffix = doc["prompt"].split("[insert]")
```

Comment:
See comment in `_make_infill_prompt` about newlines, we should probably build some prompts with DS100 script and our code to make sure they're identical